Runtime/JSDate.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace NodeApi;
+
+public readonly struct JSDate : IEquatable<JSValue>
+{
+    private readonly JSValue _value;
+
+    public static explicit operator JSDate(JSValue value) => new(value);
+    public static implicit operator JSValue(JSDate date) => date._value;
+
+    private JSDate(JSValue value)
+    {
+        _value = value;
+    }
+
+    public JSDate()
+    {
+        _value = JSValue.Global["Date"].CallAsConstructor();
```

Comment:
>JSValue.Global["Date"] [](http://example.com/codeflow?start=17&length=22)

Should we consider caching these constructors in JSContext?

Reply:
Oh, I forgot I already built a way to do that with `JSContext.Current.Import('Date')`. Updated.